main.py

```diff
@@ -1,7 +1,9 @@
 def main():
+    print("This is the real stuff")
     print("Hey")
 
```

Comment:
There is some extra whitespace here, can you fix it?

Reply:
Sure

Reply:
Ok looks good, im merging